integrations/huggingface-transformers/tutorials/token-classification/token-classification-cli.md

```diff
@@ -0,0 +1,273 @@
+
+# Modifiers:
+training_modifiers:
+  - !EpochRangeModifier
+      end_epoch: eval(num_epochs)
+      start_epoch: 0.0
+
+  - !LearningRateFunctionModifier
+      start_epoch: 0
+      end_epoch: eval(num_epochs)
+      lr_func: linear
+      init_lr: eval(init_lr)
+      final_lr: eval(final_lr)
+    
+quantization_modifiers:
+  - !QuantizationModifier
+      start_epoch: eval(qat_start_epoch)
+      disable_quantization_observer_epoch: eval(observer_epoch)
+      freeze_bn_stats_epoch: eval(observer_epoch)
+      quantize_embeddings: eval(quantize_embeddings)
+      quantize_linear_activations: 0
+      exclude_module_types: ['LayerNorm']
+      submodules:
+        - bert.embeddings
+        - bert.encoder
+        - classifier
+
+distillation_modifiers:
+  - !DistillationModifier
+     hardness: eval(distill_hardness)
+     temperature: eval(distill_temperature)
+     distill_output_keys: [logits]
+
+constant_modifiers:
+  - !ConstantPruningModifier
+      start_epoch: 0.0
+      params: __ALL_PRUNABLE__
+```
+
+</details>
+
+You can download the recipe with the following code:
+
+```python
+from sparsezoo import Model
+transfer_stub = "zoo:nlp/token_classification/obert-base/pytorch/huggingface/conll2003/pruned90_quant-none"
+download_dir = "./transfer_recipe"
+zoo_model = Model(transfer_stub, download_path=download_dir)
+recipe_path = zoo_model.recipes.default.path
+print(recipe_path)
+```
+
+### **Export to ONNX**
+
+Once you have trained your model, export to ONNX in order to deploy with DeepSparse. The artifacts of the training process are saved to your local filesystem. 
+
+Run the following to convert your PyTorch checkpoint to ONNX:
+
+```bash
+sparseml.transformers.export_onnx \
+  --model_path ./sparse_bert-token_classification_conll2003 \
+  --task text_classification
+```
+
+A `deployment` folder is created in your local directory, which has all of the files needed for deployment with DeepSparse including the `model.onnx`, `config.json`, and `tokenizer.json` files.
+
+## Sparse Transfer Learning with a Custom Dataset (WNUT_17)
+
+Beyond the Conll2003 dataset, we can also use a dataset from the Hugging Face Hub or pass via local files. Let's try an example of each for the sentiment analysis using [WNUT 17](wnut_17), which is also a NER task.
+
+For simplicity, we will perform the fine-tuning without distillation. Although the transfer learning recipe contains distillation
+modifiers, by setting `--distill_teacher disable` we instruct SparseML to skip distillation.
+
+### WNUT_17 Inspection
+
+Run the following to inspect the Rotten Tomatoes dataset.
+
+```python
+from datasets import load_dataset
+
+wnut_17 = load_dataset("wnut_17")
+print(wnut_17)
+print(wnut_17["train"][0])
+```
+
+We can see that each row dataset contains a `tokens` field which contains a list of strings representing each word the sentence and a corresponding `ner_tags` which is a list of integers representing the tag of each word in the sentence.
```

Comment:
We can't see

Reply:
We can see that each row dataset contains a tokens field which contains a list of strings representing each word in the sentence, and a corresponding `ner_tags`, which is a list of integers representing the tag of each word in the sentence.